PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,25 @@
+# PLEASE READ AND FOLLOW THE TEMPLATE! THE PR CAN BE REJECTED OTHERWISE (This line should be removed when submitting)
+
+## Description
+
+Please list all relevant dependencies in this section and provide summary of the change, motivation and context.
+
+Fixes # (issue)
+
+## Type of change
+
+- [ ] Bugfix
+- [ ] Feature
+
+## Checklist 
+
+- [ ] I have read the contribution guide: https://vaadin.com/docs-beta/latest/guide/contributing/overview/
+- [ ] I have added a description following the guideline.
+- [ ] The issue is created in the corresponding repository and I have referenced it.
+- [ ] I have added tests to ensure my change is effective and works as intended.
+- [ ] New and existing tests are passing locally with my change.
```

Comment:
Can/Should we mention the the new test should fail if the new code is removed?

Reply:
It's specified within previous point: `I have added tests to ensure my change is effective and works as intended.`

Reply:
Oh, okay. Right! 👍